File: Csharp/kids-with-the-greatest-number-of-candies.cs
public class Solution {
    public IList<bool> KidsWithCandies(int[] candies, int extraCandies) {
       
        /* using System.Linq; */
        int maxcandy = candies.Max();

        /* without linq */
        maxcandy = Int32.MinValue;
        foreach (var num in arr)
        {
            if (num > maxval)
                maxval = num;
        }
        List<bool> res = new List<bool>();
        foreach(var candy in candies){
            res.Add(candy+extraCandies>=maxcandy);
        }
        return res;
    }
}

File: javascript/kids-with-the-greatest-number-of-candies.js
var kidsWithCandies = function(candies, extraCandies) {
    
    /* Different ways of finding the max */
    /*let maxcandy = candies.reduce((a,b)=>{
        return Math.max(a,b);
    });
    
    let maxcandy = Math.max.apply(null, candies); */
    
    let maxcandy = Math.max(...candies);
    
    let boolean = new Array(candies.length);
    for (let i = 0; i < candies.length; i++) {
        boolean[i] = candies[i]+extraCandies >= maxcandy;
    }
   
    return boolean;
};